cmds/gzip/gzip.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/u-root/u-root/pkg/gzip"
+)
+
+var cmdLine = flag.CommandLine
```

Comment:
Did you try this with bb mode? I don't think bb mode handles this correctly right now.

Reply:
Yep works great, booted https://125-109016190-gh.circle-artifacts.com/0/bb_initramfs.linux_amd64.cpio.lzma from https://circleci.com/gh/bensallen/u-root/125#artifacts/containers/0:

```
% gzip -h

Usage of gzip:
  -1	Compression Level 1
  -2	Compression Level 2
  -3	Compression Level 3
  -4	Compression Level 4
  -5	Compression Level 5
  -6	Compression Level 6
  -7	Compression Level 7
  -8	Compression Level 8
  -9	Compression Level 9
  -S string
    	Specify suffix for compression (default ".gz")
  -b int
    	Set compression block size in KiB (default 128)
  -c	Write all processed output to stdout (won't delete)
  -d	Decompress the compressed input
  -f	Force overwrite of output file and compress links
  -h	Display a help screen and quit
  -k	Do not delete original file after processing
  -p int
    	Allow up to n compression threads (default 1)
  -q	Print no messages, even on error
  -t	Test the integrity of the compressed input
  -v	Produce more verbose output
wait: exit status 2
%
```

Reply:
huh. cool!